NOTES.VALGRIND

```diff
@@ -18,21 +18,21 @@ Requirements
 2. Valgrind installed on the platform
    See: http://valgrind.org/downloads/current.html
 3. OpensSSL compiled
-   See: INSTALL
+   See: [INSTALL.md](INSTALL.md)
```

Comment:
[INSTALL.md] is equivalent here (and in similar other places where the link URI and the link label are identical)

Reply:
Thanks for this hint - so I was able to remove quite some clutter from many links.